src/main/kotlin/com/openlattice/shuttle/destinations/AddressedDataHolder.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2018. OpenLattice, Inc.
```

Comment:
Really minor, but this should be 2020 now :) (also in other places)

Reply:
Lol true, will go searching